pennylane/devices/_device.py

```diff
@@ -247,7 +241,7 @@ def shots(self, shots):
                 expectation values of observables
 
         Raises:
-            DeviceError: if number of shots is less than 1
+            qml.DeviceError: if number of shots is less than 1
```

Comment:
Same suggestion for other instances of this.

```suggestion
            ~pennylane.DeviceError: if number of shots is less than 1
```